app/layout.tsx

```diff
@@ -1,19 +1,21 @@
-import type { Metadata } from "next";
+"use client"
+
+import { GoogleReCaptchaProvider } from "react-google-recaptcha-v3";
+import React, { Suspense } from "react";
 import { Montserrat_Alternates } from "next/font/google";
 import "./globals.css";
-import React, { Suspense } from "react";
 import Navigation from "../components/Navigation";
 import Footer from "@/Footer";
-import RecaptchaProvider from "@/reCAPTCHA";
 
 const montserrat = Montserrat_Alternates({ subsets: ['cyrillic'], weight: ['400', '500', '700'] });
 
-export const metadata: Metadata = {
-    title: "Skopje Paws",
-    description: "The first personal dog walking services. Not just professionals, but animal lovers too.",
-    icons: {
-        icon: '/favicon.jpg',
-    },
+const RecaptchaProvider: React.FC<{ children: React.ReactNode }> = ({ children }) => {
```

Comment:
You missed the point of this change.
We don't need a Provider that wraps the children.
Just use the GoogleCaptchaProvider where it's needed.

Reply:
On review - I think it's best if you just use the GoogleCaptchaProvider around the form since it's always gonna have the "use client" directive in it.

Something like this should do the trick
![image](https://github.com/learnhubmk/skopje-paws/assets/11035261/21daeba4-c26c-4271-8733-a240872c9d32)

Reply:
Additionally, the form tag should be around all the inputs, not just the button.

Reply:
It couldn't be just wrapped around the form, so I did this instead https://github.com/t49tran/react-google-recaptcha-v3/issues/73